### ISSUE TITLE
Publish KaiZen-OpenApi-Parser to Maven Repo

### ISSUE THREAD

**Issue by tfesenko** [RepreZen/KaiZen-OpenApi-Parser#48]
Publish KaiZen-OpenApi-Parser to Maven Repo so people could use it without compiling sources

**Comment by maybeec** [comment]
Any progress here? We would like to give it a try as swagger parser is toooooo slow. :)

**Comment by tfesenko** [comment]
Hi @maybeec , you can try our experimental Maven repo on https://bintray.com/reprezen/KaiZen-OpenApi-Parser/openapi-parser

**Comment by maybeec** [comment]
great! Thanks, we will give it a chance

**Comment by tfesenko** [comment]
The repository URL is 
```xml
        <repository>
            <id>kaizen-openapi-parser-repo</id>
            <name>KaiZen-OpenApi-Parser Repository</name>
            <url>https://dl.bintray.com/reprezen/KaiZen-OpenApi-Parser/</url>
        </repository>
```
Depency declaration as described on https://bintray.com/reprezen/KaiZen-OpenApi-Parser/openapi-parser. For the current latest versiion it's  
```xml

<dependency>
  <groupId>com.reprezen.kaizen</groupId>
  <artifactId>openapi-parser</artifactId>
  <version>0.0.1.201708021609</version>
  <type>pom</type>
</dependency>
```

**Comment by tfesenko** [comment]
@maybeec , great! Let us know what you think!

**Comment by maybeec** [comment]
So far it works quite good and a lot more performant than swagger parser. We did not found any issues however, we are not yet at the end of our implementation :)

Is there any time shedule for a release candidate or somehting like this to be published to maven? We are currently working with the repository configuration as you proposed, but it is ugly due to internal mirrors, shared configurations and so on. I would really like to see this package on maven central soon. 🥇

**Comment by maybeec** [comment]
Any progress here? Can you also start to publish source bundles?
https://bintray.com/reprezen/KaiZen-OpenApi-Parser/openapi-parser/0.0.1.201708181620#files/com%2Freprezen%2Fkaizen%2Fopenapi-parser%2F0.0.1.201708181620 does not contain any, which makes it hard to debug.
Currently, we are facing some performance issues, we would like to understand.

**Comment by maybeec** [comment]
Is this project already dead?

**Comment by tedepstein** [comment]
@maybeec , not dead, we have just been short-staffed due to vacation schedules and critical work in a few areas.  We will get to this as soon as we can, and will post back shortly with an ETA if it's likely to be more than a few days.

**Comment by tedepstein** [comment]
BTW, @tfesenko and @maybeec , I suggest we close this issue and open two new ones:
* Publish source bundles to Maven 
* Publish to Maven Central 

It seems the first one (source bundles) might be the more time-sensitive one, so you can take a better look at the performance issues. Is that right?

**Comment by tfesenko** [comment]
Hi @maybeec ! 
> Can you also start to publish source bundles?

Done! Sources and javadocs available since [0.0.1.201709121331](https://dl.bintray.com/reprezen/KaiZen-OpenApi-Parser/com/reprezen/kaizen/openapi-parser/0.0.1.201709121331/) . 

> Is there any time schedule <...> to be published to maven? 

We are looking at it. I created #67 "Publish to Maven Central" for this

**Comment by andylowry** [comment]
@tfesenko Can we close this issue? I believe it was split into #67 (complete and closed), and #49 (still in progress?).
